@navikt/core/css/darkside/guide-panel.darkside.css

```diff
@@ -1,96 +1,124 @@
 .navds-guide-panel {
-  --__ac-guide-panel-guide-size: 4rem;
+  --__axc-guide-panel-guide-size: 4rem;
+  --__axc-guide-panel-guide-padding-top: calc((var(--__axc-guide-panel-guide-size) + 1rem) + var(--ax-spacing-05));
+  --__axc-guide-panel-arrow-width: 20px;
+  --__axc-guide-panel-arrow-height: 33px;
 
   position: relative;
-  padding-top: calc(var(--__ac-guide-panel-guide-size) / 2);
-}
-
-.navds-guide {
-  background: var(--ac-guide-panel-illustration-bg, var(--a-surface-alt-3-subtle));
-  border: 2px solid var(--ac-guide-panel-border, var(--a-border-alt-3));
-  border-radius: var(--a-border-radius-full);
-  overflow: hidden;
-  position: absolute;
-  width: var(--__ac-guide-panel-guide-size);
-  height: var(--__ac-guide-panel-guide-size);
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.navds-guide svg,
-.navds-guide img {
-  height: 100%;
-  width: 100%;
-}
-
-.navds-guide-panel__content {
-  background-color: var(--ac-guide-panel-bg, var(--a-surface-default));
-  border: 2px solid var(--ac-guide-panel-border, var(--a-border-alt-3));
-  border-radius: var(--a-border-radius-large);
-  padding: var(--a-spacing-4);
-  padding-top: var(--a-spacing-12);
-}
-
-@media (min-width: 480px) {
-  .navds-guide-panel {
-    --__ac-guide-panel-guide-size: 6.25rem;
+  padding-top: var(--__axc-guide-panel-guide-padding-top);
+
+  .navds-guide {
+    background: var(--ax-bg-info-moderate);
+    border: 2px solid var(--ax-border-info);
+    border-radius: var(--ax-border-radius-full);
+    overflow: hidden;
+    position: absolute;
+    width: var(--__axc-guide-panel-guide-size);
+    height: var(--__axc-guide-panel-guide-size);
+    left: 50%;
+    transform: translate(-50%, calc(-1 * var(--__axc-guide-panel-guide-padding-top)));
+
+    svg,
+    img {
+      height: 100%;
+      width: 100%;
+    }
   }
 
   .navds-guide-panel__content {
-    padding: var(--a-spacing-8);
-    padding-top: 4.25rem;
+    position: relative;
+    background-color: var(--ax-bg-raised);
+    border: 2px solid var(--ax-border-info);
+    border-radius: var(--ax-border-radius-xlarge);
+    padding-block: var(--ax-spacing-3);
+    padding-inline: var(--ax-spacing-4);
   }
-}
 
-/* not-poster */
+  .navds-guide-panel__arrow {
+    position: absolute;
+    top: calc(
+      (
+          -1 * (var(--__axc-guide-panel-arrow-height) -
+                ((var(--__axc-guide-panel-arrow-height) / 2) - (var(--__axc-guide-panel-arrow-width) / 2)))
+        ) + 0px
+    );
+    left: calc(50% + (-1 * var(--__axc-guide-panel-arrow-width)) + var(--ax-spacing-2));
+    transform: rotate(90deg) rotateX(180deg) translateX(1.5px);
+
+    /* stylelint-disable-next-line */
+    @-moz-document url-prefix() {
```

Comment:
also not super pleased with this, if anyone wants to help debug the 1px difference on FF vs Chrome with me, I can showcase the error (I will assume it's not just me / my plugins in different browsers etc). If you remove this block, you should see it. 👀

Reply:
We could try loading 2 separate _pre transformed_ SVGs to simplify the mathematics a bit (the transforms & the left & top calculations), this _might_ get rid of slight calculation differences across browsers as well as make the calculations cleaner & easier to read. at the expense of sending an extra 625 bytes 😅.

Reply:
One issue is that we have CSS only toggling between horizontal and vertical, meaning we would have to ideally load the SVG variant directly via `url(...)` in the CSS rather than through inline html / tsx.